data/json/itemgroups/Weapons_Mods_Ammo/guns.json

```diff
@@ -442,7 +442,7 @@
       { "item": "sharps", "prob": 10, "charges-min": 0, "charges-max": 0 },
       { "item": "sks", "prob": 20, "charges-min": 0, "charges-max": 0 },
       { "item": "m1a", "prob": 30, "charges-min": 0, "charges-max": 0 },
-      { "item": "fn_ps90", "prob": 30, "charges-min": 0, "charges-max": 0 },
+      { "item": "fn_ps90", "prob": 1, "charges-min": 0, "charges-max": 0 },
```

Comment:
while the other changes make sense, are you sure your intent was to make it the rarest gun in this group by a massive margin?

Reply:
Nope it's now just one of the rarer guns. Good catch